py_client/client_api.py

```diff
@@ -303,13 +303,14 @@ def dextr3d(self, model, point_set, image_in, image_out, pad=20, roi_size='128x1
             AIAAUtils.save_result(files, image_out)
         return params
 
-    def deepgrow(self, model, foreground, background, image_in, image_out, session_id=None):
+    def deepgrow(self, model, foreground, background, current_point, image_in, image_out, session_id=None):
```

Comment:
may be u need to fix https://github.com/NVIDIA/ai-assisted-annotation-client/blob/master/py_client/test_aiaa_server.py
to support both 2D and 3D deepgrow model (with new API changes)

Reply:
I'll fix this

Reply:
fixed